x-pack/plugins/security/server/routes/authentication/common.ts

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { RouteDefinitionParams } from '..';
+
+/**
+ * Defines routes required for all authentication realms.
+ */
+export function defineCommonRoutes({ router, logger, authc, basePath }: RouteDefinitionParams) {
```

Comment:
**nit** Rather than `common` (which can often become a dumping ground over time), let's change this to be session-specific.

```suggestion
export function defineSessionRoutes({ router, logger, authc, basePath }: RouteDefinitionParams) {
```